hnn_core/extracellular.py

```diff
@@ -353,9 +385,9 @@ def smooth(self, window_len):
 
         return self
 
-    def plot(self, *, trial_no=None, contact_no=None, tmin=None, tmax=None,
-             ax=None, decim=None, color='cividis', voltage_offset=50,
-             voltage_scalebar=200, contact_labels=None, show=True):
+    def plot_lfp(self, *, trial_no=None, contact_no=None, tmin=None, tmax=None,
+                 ax=None, decim=None, color='cividis', voltage_offset=50,
+                 voltage_scalebar=200, show=True):
```

Comment:
We decided to get rid of the `contact_labels` argument at this level since it can be accessed through the Matplotlib figure instance returned by this function.